### PR TITLE
behnaz/update forex lp img base on language switcher

### PR DESCRIPTION
**Pull request by behnaz-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — APPROVED** [review]

File: src/pages/landing/ebooks/forex.js [hunk 1]
```diff
@@ -30,16 +31,27 @@ const topicsCovered = [
...
+            ...bannerImage
+        }
+        forex_ebook_inside_es: file(relativePath: { eq: "ebooks/forex-ebook-inside-es.png" }) {
             ...bannerImage
         }
     }
 `
 
 const ForexEbook = () => {
+    const { i18n } = useTranslation('home')
```
Comment (suthesan-deriv) [inline src/pages/landing/ebooks/forex.js:50]: Can we get the language from the props available in ForexEbook component instead to reduce imports?...should be available in `props.language`

Comment (sean-binary) [reply src/pages/landing/ebooks/forex.js:50]: You can also get it from `localStorage.getItem('i18n')`.

Comment (behnaz-fs) [reply src/pages/landing/ebooks/forex.js]: done thanks.

**Review by suthesan-deriv — APPROVED** [review]
LGTM

**Pull request merged**